Allow CrowdAnki export for the default 2.1.55+ exporter

aplaice opened the pull request:
Fix #169.

The old, legacy exporter (Tools > Preferences > Legacy import/export handling) had worked before this commit and now still works.

AFAICT compatibility with Anki 2.1.50+ has been maintained (2.1.50, 2.1.54 and 2.1.55 explicitly tested).

Note that in Anki 2.1.54, CrowdAnki export is only available for the old, then default (non-beta) export method, since the hook that allows easy CrowdAnki to work with the new method was AFAICT only added in 2.1.55.

aplaice commented:
I'm not too happy (messy code dealing with multiple Anki versions), but I can't think of any obvious improvements.

(Now also tested with 2.1.56!)

caldotdev commented:
Anki does not allow me to install the plugin due to an `invalid add-on manifest`

aplaice commented:
Glad that it's working!

(I also added a `manifest.json` to the new `.ankiaddon` file attached to the release, so hopefully other people won't have issues. (but I added it after your edit, so something else resolved your problem :)).)
